Util/InstallersWin/install_gtest.bat

```diff
@@ -69,12 +75,6 @@ if not exist "%GT_BUILD_DIR%" (
 cd "%GT_BUILD_DIR%"
 echo %FILE_N% Generating build...
 
-echo.%GENERATOR% | findstr /C:"Visual Studio" >nul && (
-    set PLATFORM=-A x64
```

Comment:
I share some discusion where is mentioned that buiding without `-A x64` flag casues build error in some environments:
https://github.com/carla-simulator/carla/discussions/7237

I'm not sure about why this build errors are happening. I'm worried about removing this code could introduce more build errors like the described in the shared discussion. Coudl your share your thoughts about it?

Reply:
The point is that I had build errors on my windows machine on this. Otherwise I wouldn't have changed it. But if you look more detailed onto the diff, I've just moved the code in the batch-file some lines upwards to line 40 that it is more or less identical to the other files.
But I could imagine also what the problems of others where. If you look into line 19 I've added the GENERATOR variable to this (and other scripts) with empty definitions per default. So in the batch file the GENERATOR variable wasn't defined at all if nothing is passed by the command line options. Then the original code in line 72 does somehow fail.

Therefore, I've tried to follow the "copy exactly" formula and adapted all (most?) of the batch files to
- define the GENERATOR somewhere at top BEFORE command argument parsing
- put the (full) PLATFORM definition lines relatively close to the end of the coomand line parsing.

You see this pattern in the below batch files, too. That's then part of the "harmonizing" work of this PR.